Covid/Map/SpreadMapViewController.swift

```diff
@@ -51,33 +48,50 @@ extension RegionInfo: MKAnnotation {
 final class SpreadMapViewController: UIViewController {
     @IBOutlet private var mapView: MKMapView!
 
+    let regionIdenfitier = "region"
+
     var data = [RegionInfo]() {
         didSet {
             mapView.removeAnnotations(mapView.annotations)
             mapView.addAnnotations(data)
         }
     }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        mapView.showsCompass = false
```

Comment:
preco nie?

Reply:
@anonymcek ked zobrazujeme kompas, ten je prekryty layerom, ktory ukazuje pocet chorych. vyzera to ako bug. 
Dalsia vec, tuto ta mapa nesluzi na navigovanie, ale skor ponuka prehlad poctu nakazenych na kazdy okres.
Otazne je, ci pri takomto use-case naozaj potrebujes vediet, kde je sever. Ak hej, treba poriesit bug s prekryvanim views.

Reply:
aha jasne, ok, pripadne by sme mohli `rotateEnabled = false` a nebudu to ludia tocit, len taky minor UX, vies ludia to pri zoome otocia a budu strateni

Reply:
dik za pripomienku, zapracoval som